pages/ecosystem.vue

```diff
@@ -40,9 +40,16 @@
               :target="`panel${tierName}`"
               :value="`${tierName}`"
             >
-              {{ tierName }}
+              {{ tierName }} ({{ getFilteredResultsCount(tierName) }})
```

Comment:
Could you try if this works?

```suggestion
              {{ `${tierName} (${getFilteredResultsCount(tierName)})` }}
```